neuralprophet/configure.py

```diff
@@ -98,19 +99,33 @@ class Train:
     loss_func_name: str = field(init=False)
 
     def __post_init__(self):
+        # assert either confidence interval or quantiles or is None, or both are None
+        assert self.confidence_interval is None or self.quantiles is None, "Confidence interval and quantiles cannot both be populated, one or both must be None."
+
+        # assert that confidence interval is a float between (0, 1) if not None, then use that to create the quantiles
+        if self.confidence_interval is not None:
+            assert isinstance(self.confidence_interval, float), "Confidence interval must be a float."
+            if not (0 < self.confidence_interval < 1):
+                raise ValueError("The confidence interval specified need to be a float in-between (0,1)")
+            alpha = 1 - self.confidence_interval
+            self.quantiles = [alpha/2, 1 - alpha/2]
+
+        # convert quantiles to empty list [] if still None
+        if self.quantiles is None:
+            self.quantiles = []
+
         # assert quantiles is a list type
         assert isinstance(self.quantiles, list), "Quantiles must be in a list format, not None or scalar."
 
-        # check for empty list
-        if len(self.quantiles) == 0:
-            raise ValueError("Please specify some quantile to estimate uncertainty")
+        # check if quantiles contain 0.5 and remove if so, as it will be inserted again as first index
+        if 0.5 in self.quantiles:
```

Comment:
maybe we should use `math.isclose` to compare float numbers?

Reply:
Maybe later on if this is needed.

Reply:
This is done:

```
# check if quantiles contain 0.5 or close to 0.5, remove if so as 0.5 will be inserted again as first index
self.quantiles = [quantile for quantile in self.quantiles if not math.isclose(0.5, quantile)]
```